.env.example

```diff
@@ -1 +1,2 @@
 REACT_APP_API_URL=https://target-mvd-api.herokuapp.com/api/v1
+REACT_APP_GOOGLE_MAPS_API_KEY=miapikey
```

Comment:
```suggestion
REACT_APP_GOOGLE_MAPS_API_KEY=<API_KEY>
```

Reply:
I was using npm 16 instead of 14 and it had conflicts when installing google-maps-react, so I backed down to v14 and it worked